gateway/src/main/resources/config.yml.template

```diff
@@ -1,22 +1,24 @@
 requestRouter:
   port: 8080
-  name: Proxly
+  name: prestoRouter
   cacheDir: /var/log/prestoproxy/cache
   historySize: 1000
 
 backends:
   - localPort: 8081
     name: presto1
     proxyTo: http://presto1.lyft.com
+    routingGroup: adhoc
```

Comment:
Is this necessary ? Isnt the default adhoc

Reply:
its a config template for reference. User may opt out to set this field explicitly.